bears/installation_tool.py

```diff
@@ -0,0 +1,161 @@
+import argparse
+import importlib
+import re
+import subprocess
+import sys
+
+from coala_utils.string_processing.StringConverter import StringConverter
+from coala_utils.Question import ask_question
+from coalib.collecting.Collectors import get_all_bears_names
+from coalib.collecting.Importers import iimport_objects
+from coalib.misc.Shell import call_without_output
+from prompt_toolkit import prompt
+from prompt_toolkit.contrib.completers import WordCompleter
+from pygments.lexers import SqlLexer
+from pyprint.ConsolePrinter import ConsolePrinter
+
+
+def install_pip_package(package_name):
+    """
+    Uses call to install a PyPi package.
+
+    :param package_name: The package to be installed.
+    """
+    print('To install the dependencies correctly, we will need administrator '
+          'rights from you.')
```

Comment:
I *think* `root permission` is the correct word, not `admin rights`. Not sure, confirm with others.